src/main/resources/application.properties

```
@@ -0,0 +1,16 @@
+precursor.tolerance=0.5
+fragment.tolerance=0.5
+
+
+# Clustering execution parameters
+threshold.start=0.999F
```

This gives the best results:

```suggestion
threshold.start=1.0F
```